jsapp/scss/components/_kobo.modal.scss

```diff
@@ -484,6 +484,281 @@
   }
 }
 
+.form-modal--translation-settings {
+  .form-view__cell--translation-note {
+    padding-bottom: 20px;
+  }
+  .form-view__cell--translation {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-top: 1px solid $divider-color;
+    padding: 6px 0px;
+  }
+
+  .form-view__cell--translation-actions {
+    min-width: 150px;
+    text-align: right;
+  }
+
+  .form-view__cell--add-language {
+    padding-top: 20px;
+    text-align: right;
+    border-top: 1px solid $divider-color;
+  }
+
+  .form-view__cell--add-language-form,
+  .form-view__cell--update-language-form {
+    background: #F8F8F8;
+    padding: 15px;
+    position: relative;
+
+    .form-view__link--close {
+      position: absolute;
+      right: 0px;
+      top: 6px;
+
+      i {
+        font-size: 24px;
+      }
+    }
+  }
+
+  .form-view__cell--add-language-form {
+    margin-top: 20px;
+  }
+
+  .form-view__cell--add-language-fields {
+    display: flex;
+    justify-content: space-between;
+
+    .form-view__cell {
+      min-width: 100px;
+
+      &--lang-code {
+        max-width: 50px;
+      }
+
+      input {
+        padding-left: 5px;
+        max-width: 95%;
+      }
+
+      button {
+        margin-left: 10px;
+        margin-top: 5px;
+      }
+    }
+  }
+}
+
+.form-modal--translation-table {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: 0;
+
+  .form-modal__item--translation-table--container {
```

Comment:
Nitpicking: isn't this invalid BEM selector? I would go with `.form-modal__item--translation-table-container`